src/index.html

```diff
@@ -185,5 +189,12 @@
         }
       });
     </script>
+    <script>
+      setTimeout(()=>{
```

Comment:
Would it be possible to remove the logic from the sample app? Don't think it's necessary to include this in the sample for the component library